components/browser/storage-sync/docs/metrics.md

```diff
@@ -0,0 +1,48 @@
+<!-- AUTOGENERATED BY glean_parser.  DO NOT EDIT. -->
+
+# Metrics
+This document enumerates the metrics collected by this project.
+This project may depend on other projects which also collect metrics.
+This means you might have to go searching through the dependency tree to get a full picture of everything collected by this project.
+Sorry about that.
+
+# Pings
+
+ - [bookmarks_sync](#bookmarks_sync)
+ - [history_sync](#history_sync)
+
+
+## bookmarks_sync
+A ping sent for every bookmarks sync. It doesn't include the `client_id` because it reports a hashed version of the user's Firefox Account ID.
+
+The following metrics are added to the ping:
+
+| Name | Type | Description | Data reviews | Extras | Expiration |
+| --- | --- | --- | --- | --- | --- |
+| bookmarks_sync.failure_reason |[labeled_string](https://mozilla.github.io/glean/book/user/metrics/labeled_strings.html) |Records bookmark sync failure reasons.  |[1](https://github.com/mozilla-mobile/android-components/pull/3092)||never |
```

Comment:
Great catch! This is indeed a bug, we should document labels. Since this is done by the glean_parser fixing this requires to release a new version. Once we do so, changes will be automatically picked up by all the repos. I've filed [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1578345) for fixing labels.